Rivet/Functions/Invoke-Rivet.ps1

```diff
@@ -116,10 +125,30 @@ Found no databases to migrate. This can be a few things:
             return
         }
 
+        $confirmDropDatabase = $false
+        if( $DropDatabase -and -not $Force)
+        {
+            $confirmQuery = "Using the `DropDatabase` switch will drop the database(s) for the current environment. Do you want to proceed?"
+            $confirmCaption = "Drop the following database(s)? {0}" -f ($settings.Databases | Select-Object -Property Name | Out-String)
+            $confirmDropDatabase = $PSCmdlet.ShouldContinue( $confirmQuery, $confirmCaption )
+        }
+
         foreach( $databaseItem in $settings.Databases )
         {
             $databaseName = $databaseItem.Name
             $dbMigrationsPath = $databaseItem.MigrationsRoot
+

```

Comment:
First, determine if any of the databases even exist. If none of the databases exist, then return with no errors. Otherwise, drop any databases that exist.